src/test/java/org/broadinstitute/hellbender/tools/exome/SegmentUtilsUnitTest.java

```diff
@@ -22,10 +22,10 @@
 
     @Test
     public void testReadUncalledSegments() {
-        final File TEST_DIR = new File("src/test/resources/org/broadinstitute/hellbender/tools/exome/caller");
-        final File TEST_SEGMENTS = new File(TEST_DIR,"segments.tsv");
+        final File testDir = new File("src/test/resources/org/broadinstitute/hellbender/tools/exome/caller");
```

Comment:
I guess it was like this originally because it is only used in this test, but done.